src/components/inventory/editView/GeneralSettings.tsx

```diff
@@ -33,6 +34,12 @@ export default function GeneralSettings({ locked }: GeneralSettingsProps) {
     }));
   };
 
+  const getValueWithBackup = (value?: string | number) => {
+    const backedUpValue = value || 'undefined';
```

Comment:
Should this be "undefined"? 
Shouldn't it be more "normal language" and added to i18n?